docs/pages/configuration/pipelines/functions/select_pod.mdx

```diff
@@ -2,3 +2,48 @@
 title: Function select_pod
 sidebar_label: select_pod
 ---
+
+This function is used to select the container (pod) to be used in the pipeline when it is run.
+
+## Arguments
+
+
+### `--image-selector`
+
+The image selector used to select the container
+
+### `--label-selector`
+
+The label selector used to select the container
+
+### `--container`
+
+The container to be used.
+
+### `--namespace`
+
+The namespace to be used.
+
+## Usage
+
+```
+select_pod [--image-selector|--label-selector]
+```
+
+
+## Flags
+
+By default, flags are disabled. If a flag listed below is enabled, the following action will be taken.
+
+### `--disable-wait`
+
+The pod will be selected, regrdless of whether the container is ready.
+
+### `--timeout`
+
+The duration of time the before the request times out. The default is 5 minutes.
```

Comment:
What "request"? The `select_pod` command?

Reply:
fixed